util/types/mydecimal.go

```diff
@@ -726,8 +745,9 @@ func (d *MyDecimal) Round(to *MyDecimal, frac int) (err error) {
 	toIdx := wordsInt + wordsFracTo - 1
 	if frac == wordsFracTo*digitsPerWord {
 		doInc := false
-		switch roundDigit {
-		case 0:
+		switch roundMode {
```

Comment:
Please swith `roundDigit` like:
```
switch roundDigit {
case 0:
case 5:
case 10:
}
```
The number 0, 5, 10 is used literally the digit number, do not need any constant representation.